package.json

```diff
@@ -61,9 +64,11 @@
     "css-loader": "^5.2.7",
     "css-minimizer-webpack-plugin": "^3.4.1",
     "cssnano": "^5.0.17",
+    "cypress": "^7.4.0",
     "dotenv-webpack": "^7.1.0",
     "eslint": "^8.10.0",
     "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-cypress": "^2.11.2",
```

Comment:
same question as above, latest seems to be 2.12.1

Reply:
@jomarko no reason, right now. I was not sure to put the latest version (2.12.1).